### PR TITLE
Set the compilation target to JDK 11

### PR DESCRIPTION
**Pull request by m1ngyuan**

### CODE REVIEW

**Review by gsmet — APPROVED** [review]
Good idea!

File: build-parent/pom.xml [hunk 1]
```diff
@@ -874,7 +874,7 @@
                                                     maven-antrun-plugin
                                                 </artifactId>
                                                 <versionRange>
-                                                    [1.8,)
+                                                    [11,)
```
Comment (famod) [inline build-parent/pom.xml:877]: This doesn't look right.

Comment (m1ngyuan) [reply build-parent/pom.xml:877]: Yes, thank you for the reminder.

**Comment by gsmet** [comment]
@Sanne hmmm, maybe we should have waited for the CI run to complete?

**Comment by Sanne** [comment]
@gsmet I did.. but the PR was push-forced just about the same time :/

**Comment by gsmet** [comment]
This PR broke the build, I'm working on fixing it.

**Comment by gsmet** [comment]
https://github.com/quarkusio/quarkus/pull/18067

**Comment by Sanne** [comment]
I'm sorry :/ Thanks!

**Pull request merged**